### DISCUSSION TITLE
Can't translate english to vietnamese

### DISCUSSION THREAD

**Discussion by hieu-delta** [N0I0C0K/PowerTranslator#18] [Q&A] [answered]
I've install PowerTranslator, but I can't translate English words to Vietnamese. 
Vietnamese language code is 'vi', but when I type " ^song -> vi ", nothing happen. How can I fix it ?

**Comment by N0I0C0K** [comment, marked as answer]
Hello!  I just went to check the translation interface and found an unfortunate news, although the official offer of English to Vietnamese language translation options, but the translation interface does not actually return any results! **This problem is caused by translate interface**. I will later add a translation interface with good support for most languages. Hope this is helpful to you!

**Comment by N0I0C0K** [comment]
Hi! Recently I noticed that the translation from English to Vietnamese seems to be OK, you can check it！

![image](https://github.com/N0I0C0K/PowerTranslator/assets/30338746/7d90c4c4-a2bb-482b-a6e0-7f80b14761e9)

**Reply by hieu-delta** [reply]
thanks you so much
